releases/release-7.1.0.md

```diff
@@ -23,28 +23,48 @@ In v7.1.0-LTS, the key new features and improvements are as follows:
 </thead>
 <tbody>
   <tr>
-    <td rowspan="2">Scalability and Performance</td>
-    <td><a href="https://docs.pingcap.com/tidb/dev/sql-non-prepared-plan-cache" target="_blank">Session-level plan cache for non-prepared statements</a> (experimental)</td>
-    <td>Support automatically reusing plan cache at the session level to remove query planning time, reducing query time for repeat SQL patterns without manually setting prepare statements in advance.</td>
+    <td rowspan="3">Scalability and Performance</td>
+    <td><a href="https://docs.pingcap.com/tidb/dev/tiflash-disaggregated-and-s3" target="_blank">TiFlash decouples storage and compute, and adds S3 shared storage (experimental)</a></td>
+    <td>TiFlash introduces a cloud-native architecture as an option:
+Disaggregates TiFlash's compute and storage, which is a milestone for elastic HTAP resource utilization.
+Introduces S3-based storage engine, which can provide shared storage at a lower cost.
+    </td>
+  </tr>
+  <tr>
+    <td><a href="https://docs.pingcap.com/tidb/dev/system-variables#tidb_store_batch_size" target="_blank">TiKV supports batch aggregating data requests</a></td>
```

Comment:
@Oreoxmt This URL is "/dev/" instead of a hardcoded version number, which is unlike previous docs versions. Will this work if we just replace the "dev" with "7.1"?

Reply:
@samhld https://docs.pingcap.com/tidb/v7.1 is not available (404 Not Found) before the release day. I will create a new PR to update all links in the highlight table from `/dev` to `/v7.1` after https://docs.pingcap.com/tidb/v7.1 is available.